tern/analyze/common.py

```diff
@@ -218,10 +220,29 @@ def convert_to_pkg_dicts(pkg_dict):
         a_pkg = {}
         for key, value in new_dict.items():
             a_pkg.update({key: value[index]})
+        if a_pkg.get('files', False):
+            add_package_files(a_pkg)
         pkg_list.append(a_pkg)
     return pkg_list
 
 
+def add_package_files(a_pkg):
+    """Convert a list of raw file strings in a package
```

Comment:
Can you add here in the description what the "raw file string" is supposed to look like?